### PR TITLE
refactor: Move streamVerifier to util/execpool/stream (phase 2)

### PR DESCRIPTION
**Pull request by algonautshant**
Move streamVerifier to util/execpool
Add utint tests to util/execpool/stream 
update the call sites to this interface

### CODE REVIEW

File: util/execpool/stream_test.go [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+			require.Equal(t, mockJobs[i].batchSize-1, mockJobs[i].batchOrder)
+		}
+		if mockJobs[i].returnError != nil {
+			require.Nil(t, mockJobs[i].returnError)
+		}
+		require.True(t, mockJobs[i].processed)
+	}
+}
+
+// TestNoInputYet let the servicd start and get to the timeout without any inputs
```
Comment (algorandskiy) [inline util/execpool/stream_test.go:199]: servicd -> service

**Review by algorandskiy — COMMENTED** [review]
LGTM, indeed a code move and test

File: util/execpool/stream.go [hunk 2]
```diff
@@ -38,6 +37,8 @@ var ErrShuttingDownError = errors.New("not processed, execpool service is shutti
 // for processing before waitForNextJobDuration.
 const waitForNextJobDuration = 2 * time.Millisecond
 
+const txnPerWorksetThreshold = 32
```
[the remark this answers is not in the document]
Comment (algonautshant) [reply util/execpool/stream.go:40]: Used in stream.go:
```
125:			if numberOfJobsInCurrent > txnPerWorksetThreshold {
```

**Review by algorandskiy — APPROVED** [review]

**Review by iansuvak — APPROVED** [review]
Both tests and code-move look good

**Pull request merged**